### PR TITLE
New vote.py + noprompt addition to prompt_pager() in common.py

### PR DESCRIPTION
**Pull request by ericolito**
New additions to vote.py:
* Now utilizes prompt_pager.
* The result output looks great on terminals with fewer columns than 80.
* unicode fixes.
* etc..

New additions to common.py
* a "noprompt" option that disables the press enter prompt.

### CODE REVIEW

File: x84/default/common.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def display_banner(filepattern, encoding=None, vertical_padding=0):
     return line_no + vertical_padding
 
 
-def prompt_pager(content, line_no=0, colors=None, width=None, breaker=u'- '):
+def prompt_pager(content, line_no=0, colors=None, width=None, breaker=u'- ', noprompt=False):
```
Comment (jquast) [inline x84/default/common.py]: prompt ... noprompt i would think this means "do nothing, then"

this absolutely requires a docstring -- see below 4 lines where ":param iterable content: ....", please add.

File: x84/default/common.py [hunk 2]
```diff
@@ -113,12 +113,13 @@ def show_breaker():
...
-    echo(u'\r\n')
-    if term.width > 80:
-        echo(term.move_x((term.width // 2) - 40))
-    echo(u'Press {enter}.'.format(
-        enter=colors['highlight'](u'return')))
-    inp = LineEditor(0, colors=colors).read()
+    if noprompt == False:
+        echo(u'\r\n')
+        if term.width > 80:
+            echo(term.move_x((term.width // 2) - 40))
```
Comment (jquast) [inline x84/default/common.py]: just use max(0, ...) and forget the if statements

File: x84/default/common.py [hunk 3]
```diff
@@ -113,12 +113,13 @@ def show_breaker():
                 echo(term.move_up() + term.clear_eol)
 
     show_breaker()
-    echo(u'\r\n')
-    if term.width > 80:
-        echo(term.move_x((term.width // 2) - 40))
-    echo(u'Press {enter}.'.format(
-        enter=colors['highlight'](u'return')))
-    inp = LineEditor(0, colors=colors).read()
+    if noprompt == False:
```
Comment (jquast) [inline x84/default/common.py]: would rather see "if no prompt: return"

**Comment by ericolito** [comment]
and that fucking landscape-bot can go to where the chili-peppers grow.

**Pull request merged**